corehq/apps/app_manager/util.py

```diff
@@ -752,3 +756,16 @@ def extract_instance_id_from_nodeset_ref(nodeset):
     if nodeset:
         matches = re.findall(r"instance\('(.*?)'\)", nodeset)
         return matches[0] if matches else None
+
+
+def does_app_have_mobile_ucr_v1_refs(app):
+    re_mobile_ucr_v1_all_refs = re.compile(MOBILE_UCR_V1_ALL_REFERENCES)
+    for form in app.get_forms():
+        # The second condition should always be False if the first one is
+        # but just as a precaution we'll check for it
+        if (
+            MOBILE_UCR_V1_FIXTURE_IDENTIFIER in form.source
+            or re_mobile_ucr_v1_all_refs.search(form.source)
+        ):
```

Comment:
nit: I assume you want to keep this explicit/clean and so you are using the if block instead of doing
```
return (MOBILE_UCR_V1_FIXTURE_IDENTIFIER in form.source or re_mobile_ucr_v1_all_refs.search(form.source))
```
I like explicit.

Reply:
I put this in a conditional since we are looping through all the forms of an app to check if there are any V1 refs. Changing this to return right away would mean only the first form will get checked.

Reply:
Not sure I understand. We want this method to return True if any form has the matching reference right?

If its written in a single line, it should not change how the method functions.

Reply:
offlined.

single link change was not the correct thing here.